code/game/objects/structures/crates_lockers/crates/gear_loadout.dm

```diff
@@ -755,6 +755,50 @@
 	new /obj/item/ammo_magazine/c45m(src)
 	new /obj/item/ammo_magazine/c45m(src)
 
+/obj/structure/closet/crate/secure/gear_loadout/ceres_lance/fill()
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/under/dress/lance_dress(src)
+	new /obj/item/clothing/under/dress/lance_dress/male(src)
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/under/dress/lance_dress(src)
+	new /obj/item/clothing/under/dress/lance_dress/male(src)
+	new /obj/item/rig/bunker(src)
+	new /obj/item/clothing/suit/space/void/lancer(src)
+	new /obj/item/clothing/head/helmet/space/void/lancer(src)
+	new /obj/item/clothing/suit/space/void/lancer(src)
+	new /obj/item/clothing/head/helmet/space/void/lancer(src)
+	new /obj/item/clothing/suit/space/void/lancer(src)
+	new /obj/item/clothing/head/helmet/space/void/lancer(src)
+	new /obj/item/clothing/suit/space/void/lancer(src)
+	new /obj/item/clothing/head/helmet/space/void/lancer(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/gun/energy/rifle/ionrifle(src)
+	new /obj/item/gun/energy/rifle/pulse(src)
+	new /obj/item/gun/energy/rifle/pulse(src)
+	new /obj/item/gun/energy/pulse/pistol(src)
+	new /obj/item/gun/energy/pulse/pistol(src)
+	new /obj/item/gun/energy/pulse/pistol(src)
+	new /obj/item/gun/energy/pulse/pistol(src)
```

Comment:
```/obj/item/projectile/beam/pulse
	name = "pulse"
	icon_state = "u_laser"
	damage = 50
	armor_penetration = 50
```

Yeah let's not add a guaranteed 50 damage beam to normal gameplay please. Remove the pulse rifles and pistols.